src/main/java/com/crafter/springboottechiteasy/controllers/TelevisionController.java

```diff
@@ -2,56 +2,54 @@
 
 import com.crafter.springboottechiteasy.exceptions.RecordNotFoundException;
 import com.crafter.springboottechiteasy.exceptions.RequirementsNotMetException;
+import com.crafter.springboottechiteasy.models.Television;
+import com.crafter.springboottechiteasy.repositories.TelevisionRepository;
+import com.crafter.springboottechiteasy.services.TelevisionService;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/televisions")
 public class TelevisionController {
 
+    private TelevisionService televisionService;
+
+    public TelevisionController(TelevisionService televisionService) {
+        this.televisionService = televisionService;
+    }
+
     @GetMapping
-    public ResponseEntity<Object> getAllTVs() {
-        return ResponseEntity.ok("television");
+    public ResponseEntity<List<Television>> getAllTVs() {
+        return ResponseEntity.ok(televisionService.getAllTelevisions());
     }
 
     @GetMapping("/{id}")
-    public ResponseEntity<Object> getOneTV(@PathVariable int id) {
-        if (id == 0) { // determine if id exists
-            throw new IndexOutOfBoundsException();
-        } else {
-            return ResponseEntity.ok(id);
-        }
+    public ResponseEntity<Television> getOneTV(@PathVariable Long id) {
+        return ResponseEntity.ok(televisionService.getOneTelevision(id));
     }
 
     @PostMapping("/add")
```

Comment:
Je maag geen woordjes als "add" gebruiken in je path. Het feit dat het hier om een PostMapping gaat, brengt al die boodschap over. We willen geen onnodige woordjes in REST URLs, de enige uitzondering zijn "controller woordjes", maar dan alleen in specifike situaties, zoals wanneer je de "isAvailable" boolean van een specifieke televisie van True naar False wilt zetten na een verkoop via het endpoint "localhost:8080/televisions/{id}/availability"